Makefile

```
@@ -6,6 +6,12 @@
 
 NODE_BIN=$(CURDIR)/node_modules/.bin
 
+WORKING_DIR := edx_credentials_themes
+EXTRACT_DIR := $(WORKING_DIR)/conf/locale/en/LC_MESSAGES
+EXTRACTED_DJANGO := $(EXTRACT_DIR)/django-partial.po
+EXTRACTED_TEXT := $(EXTRACT_DIR)/django.po
```

`text` is used in the context of XBlocks and we're deprecating effectively the use of `text.po` as part of OEP-58.

Please rename to reflect proper naming -- in case these variables are still needed.

```suggestion
EXTRACTED_DJANGO_PARTIAL := $(EXTRACT_DIR)/django-partial.po
EXTRACTED_DJANGO := $(EXTRACT_DIR)/django.po
```